src/formly/types/templates/attachment.tpl.html

```diff
@@ -19,11 +19,16 @@
 
     </table>
 
-    <button type="file" class="btn btn-primary btn-file" ngf-select="upload($file)" ng-disabled="status != STATUS.normal"
-            ngf-multiple="false" ng-show="model[options.key] == null">
+    <button type="file" class="btn btn-primary btn-file"
+            ngf-select="upload($file)"
+            ng-disabled="status != STATUS.normal"
+            ngf-multiple="false"
+            ngf-pattern="'{{acceptedExtensions}}'"
```

Comment:
This is not how it should be done.
You make conversion to string so it can be converted back into expression?
`ngf-pattern="acceptedExtensions"`
Would not work?